### PR TITLE
GP2-3164 Security vulnerabilities

### PR DESCRIPTION
**Pull request by bobmeredith**
Upgrade to fix the security vulnerabilities in great-cms.
Main thing is a bump of webpack to version 5.
Sadly, the compiled css and js don't match the old ones - which I guess is not that surprising. Should check that this works for you for both build and watch.
FYI works for me on node 13

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-3164
- [ ] Jira ticket has the correct status.
- [ ] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [ ] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [x] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: react-components/webpack.config.js [hunk 1]
```diff
@@ -79,39 +82,48 @@ module.exports = {
       },
       {
         test: /\.(jpg|png|gif|jpeg|svg)$/,
-        loader: 'url-loader?limit=10000&name=img/[name].[ext]',
+        use: [
+          {
+            loader: 'url-loader',
+            options: { limit: '10000', name: 'img/[name].[ext]' },
```
Comment (froddd) [inline react-components/webpack.config.js:88]: Updating great-styles I found the name needed to be `./img/[name].[ext]` to result in unchanged compiled CSS but I guess that should also work! May update that other PR if we're happy with slight change.

File: react-components/webpack.config.js [hunk 2]
```diff
@@ -79,39 +82,48 @@ module.exports = {
...
-    ]),
+    new CopyWebpackPlugin({
+      patterns: [
+        { from: './node_modules/great-styles/static/images', to: 'images' },
+        {
+          from: './node_modules/great-styles/static/fonts',
+          to: '../../core/static/fonts/',
+        },
+        // copies the images to core/static only if not present. This avoids
+        // the svg files showing up in diff every time a new build occurs
```
Comment (froddd) [inline react-components/webpack.config.js:115]: If images aren't copied anymore let's remove this comment, confused me a bit!

**Review by froddd — COMMENTED** [review]
LGTM, one slightly confusing leftover comment to get rid of!

File: react-components/webpack.config.js (hunk 2 above)
Comment (bobmeredith) [reply react-components/webpack.config.js:115]: Fair point

**Review by froddd — APPROVED** [review]
All good!

**Pull request merged**